### ISSUE TITLE
Better Bitwarden interface

### ISSUE THREAD

**Issue by bluethumb** [minbrowser/min#1306]
I have to use bitwardens website along with Min, because I can not see my notes and some password do not come true
Bitwarden or Min problem I do not know.

**What solution would you like?**
Better menu/Interface for bitwarden ( maybe 1pass too, I do not use it )

**Are there any alternatives you've considered?**
Well, as I do now but it is a bit of a pain !!

**Comment by PalmerAL** [comment]
Can you give examples of sites where your passwords aren't shown?

**Comment by bluethumb** [comment]
I have alot of (sub)password for a url basslab.com  and Min/bitwarden gives me a long list and it is not possible to find the correct name/password in the list because it only shows login name and no other info. The nativ app in Firefox have more info to discover the correct login/passwor. Also it is not possible to use bitwardens "safenotes"

**Comment by PalmerAL** [comment]
What information does Firefox show that helps you find the password?

**Comment by Syndamia** [comment]
I'm pretty sure he is talking about how with the Bitwarden extension (and the builtin password management) you can just go and browse all of the passwords, notes, etc. you have stored, no matter what site you are on.

![image](https://user-images.githubusercontent.com/46843671/98335389-856b5680-200d-11eb-8030-d55246e694a9.png)
*Here I'm on a site called shkolo.bg, but from the Bitwarden extension I can look up my GitHub credentials if I want*

**Comment by bluethumb** [comment]
Hmm I do not have the Bitwarden dropdown menu I have this (see picture)

![min_bitwarden](https://user-images.githubusercontent.com/1456728/98439155-2cc0ba00-20f0-11eb-9d1e-54fc3bd8d04a.png)

**Comment by bluethumb** [comment]
Also on Aliexpress Bitwarden do not show up ( after login to Bitwarden)
![alie](https://user-images.githubusercontent.com/1456728/98515191-fb202e00-226a-11eb-9e68-36d9e5af4e5a.png)

**Comment by PalmerAL** [comment]
Regarding the first issue, are all of those logins you've saved for that site?

I guess we could make the list scrollable, but I'm not sure I see what else we could include. What does this look like in Firefox?

**Comment by bluethumb** [comment]
Yes I get a long list but can not find the right login/passw in that list. If I use bitwardeb app in FF it gives me the correct login/passw and a long list

**Comment by bluethumb** [comment]
Updated to 1.17.3-2 still same problems, maybe it would help the overview  if you got an "headline" and a login name see pict. 
But it does not solve the issue with Aliexpres
![1609361866](https://user-images.githubusercontent.com/1456728/103380827-4f68c100-4aea-11eb-948d-763b0a4580d8.png)
s
